homeassistant/components/prometheus/__init__.py

```diff
@@ -169,6 +169,8 @@ def _handle_attributes(self, state):
                 metric.labels(**self._labels(state)).set(value)
             except ValueError:
                 pass
+            except TypeError:
```

Comment:
Since we already have a pass block, please consider merging them.

Reply:
> Since we already have a pass block, please consider merging them.

Done